lib/utils.go

```diff
@@ -3,16 +3,18 @@ package lib
 import (
 	"encoding/base64"
 	"encoding/xml"
+	"errors"
 	"fmt"
+	"strconv"
 	"strings"
 
 	"github.com/segmentio/aws-okta/lib/saml"
 	"golang.org/x/net/html"
 )
 
-//TODO: Move those functions into saml package
+func GetAssumableRolesFromSAML(resp *saml.Response) (saml.AssumableRoles, error) {
```

Comment:
If possible, can you restore the function `GetRoleFromSAML` and have it call these two methods? We should avoid breaking API compatibility

Reply:
Understood, I'll try to make the change this week.

Reply:
@nickatsegment `GetRoleFromSAML` restored.